_posts/2017-12-18-XPDays-Benelux-2017.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: post
+authors: [derya_duru]
+title: 'XP Days Benelux 2017'
+image: /img/xpdays-benelux-2017/XPDays-Benelux-2017.png
+tags: [XP Days Benelux, Agile, Scrum, Scrum Master, Gamification, Integral Theory, Art of Hosting, Facilitation, Coaching]
+category: Conference
+comments: true
+---
+
+> [XP Days Benelux](http://www.xpdaysbenelux.nl){:target="_blank"} is a two day conference on agile software development for and by practitioners.
+In this post, I will go over some talks and sessions I attended and participated in.
+
+<div class="the-toc">
+
+  <h1 class="the-toc__heading">Table Of Contents</h1>
+
+  <ol class="the-toc__list">
+      <li><a href="#the-8-stances-of-a-scrum-master---barry-overeem" title="The 8 Stances of a Scrum Master - Barry Overeem">The 8 Stances of a Scrum Master - Barry Overeem</a></li>
+      <li><a href="#do-not-deal-with-resistance---remi-armand-collaris--linda-dorlandt" title="Do Not Deal with Resistance! - Remi-Armand Collaris & Linda Dorlandt">Do Not Deal with Resistance! - Remi-Armand Collaris & Linda Dorlandt</a></li>
+      <li><a href="#motivate-your-team-with-gamification---jean-jacques-courtens" title="Motivate Your Team with Gamification - Jean-Jacques Courtens">Motivate Your Team with Gamification - Jean-Jacques Courtens</a></li>
+      <li><a href="#an-integral-view-on-agile---frederik-vannieuwenhuyse--johannes-schartau" title="An Integral View on Agile - Frederik Vannieuwenhuyse & Johannes Schartau">An Integral View on Agile - Frederik Vannieuwenhuyse & Johannes Schartau</a></li>
+      <li><a href="#the-art-of-hosting-conversations-that-matter---johan-decoster--jef-cumps" title="The Art of Hosting Conversations that Matter - Johan Decoster & Jef Cumps">The Art of Hosting Conversations that Matter - Johan Decoster & Jef Cumps</a></li>
+    </ol>
+
+</div>
+
+****
+
+### The 8 Stances of a Scrum Master - Barry Overeem
+
+<span class="image left"><img class="p-image" alt="Barry Overeem" src="/img/xpdays-benelux-2017/Barry-Overeem.jpg"></span>
+[Barry Overeem](https://twitter.com/barryovereem){:target="_blank"} is a freelance Scrum Master and Professional Scrum Trainer at Scrum.org. He’s an active member of the Scrum community and shares his insights and knowledge by speaking at conferences, facilitating workshops and writing blog posts.
+
+It is so common for a Scrum Master to get assigned tasks that aren't actually tasks a Scrum Master should be performing. The job description of a Scrum Master comes with whole other tasks than a lot of people assume, sometimes even the Scrum Master him or herself.
```

Comment:
People don't expect the actual job content of a Scrum Master to contain such a wide array of tasks. Often times these tasks are even missed or forgotten by Scrum Masters themselves.